### PR TITLE
Add two new terms, change organization.

### PR DESCRIPTION
**Pull request by luciansmith**
* Add 'ODE Solver', and move 'CVODE Solver' and 'Livermore solver' to be children of it.
* Add 'parameters for' for specifying more-specific parameters for more-specific algorithms.

### CODE REVIEW

**Comment by luciansmith** [comment]
This is my attempt to implement the suggestions from HARMONY and described at

https://groups.google.com/g/sed-ml-discuss/c/nIJOYsPGEag

However, there may well be KiSAO conventions I broke, so if someone could take a look, that'd be great.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Install pip and setuptools
         run: |
-          python -m pip install --upgrade pip
+          python -m pip install --upgrade pip==23.0.1
```
Comment (matthiaskoenig) [inline .github/workflows/ci.yml:34]: This should not be pinned!

File: kisao.owl [hunk 2]
```diff
@@ -14740,6 +14740,135 @@ This method only involves evaluations of f. This method is suitable for non-stif
...
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.biomodels.net/kisao/KISAO#KISAO_0000259"/>
+                <owl:maxQualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:maxQualifiedCardinality>
+                <owl:onClass rdf:resource="http://www.biomodels.net/kisao/KISAO#KISAO_0000571"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <dc:created rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2023-05-19</dc:created>
+        <dc:creator xml:lang="en">lps</dc:creator>
+        <isOrganizational rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</isOrganizational>
+        <rdfs:label>ODE Solver</rdfs:label>
```
Comment (matthiaskoenig) [inline kisao.owl:14841]: I think the lang is missing:
```
<rdfs:label xml:lang="en">ODE solver</rdfs:label>
```
Also all labels are lowercase besides special words, all other words lowercase, i.e. "ODE Solver" -> "ODE solver"

**Review by matthiaskoenig — CHANGES_REQUESTED** [review]
Looks good, see minor suggestions in review. I could load and view it in Protege.
Perhaps add also the `Euler method` and `Runga-Kuta method` as children for `ODE solver`. These are often used by tools which do not use CVODE.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (luciansmith) [reply .github/workflows/ci.yml:34]: Every version of pip past 23.0.1 will break all of our builds, here and on biosimulations.  We need to fix that, but it's a much larger project.

**Comment by luciansmith** [comment]
Both Euler and Runge Kutta are themselves children of the 'one-step method' group:  should I move the entire group to be children of ODE Solver?  Here's the whole list, with the hierarchy missing because it didn't cut-and-paste properly, but at least everything is there:

'one-step method'
'Euler method'
'Euler backward method'
'Euler forward method'
'Euler-Maruyama method'
'Richardson extrapolation based method'
'Bader-Deuflhard method'
'Bulirsch-Stoer algorithm'
'Runge-Kutta based method'
'Butcher-Kuntzmann method'
'embedded Runge-Kutta method'
'Dormand-Prince 8(5,3) method'
'embedded Runge-Kutta 5(4) method'
'Cash-Karp method'
'Numerical Recipes in C "quality-controlled Runge-Kutta" method'
'Dormand-Prince method'
'Fehlberg method'
'hierarchical Fehlberg method'
'Higham-Hall method'
'embedded Runge-Kutta Prince-Dormand (8,9) method'
'Merson method'
'Verner method'
'Zonneveld method'
'explicit fourth-order Runge-Kutta method'
'explicit Runge-Kutta method of order 3(2)'
'Gauss-Legendre Runge-Kutta method'
'Gill method'
'Hammer-Hollingsworth method'
'Heun method'
IMEX
'implicit 4th order Runge-Kutta method at Gaussian points'
'implicit midpoint rule'
'Lobatto method'
'midpoint method'
'modified midpoint method'
'partitioned Runge-Kutta method'
'Radau method'
'Rosenbrock method'
'Numerical Recipes in C "stiff" Rosenbrock method'
'semi-implicit midpoint rule'
'stochastic Runge-Kutta method'
'stochastic second order Runge-Kutta method'

**Comment by matthiaskoenig** [comment]
Makes sense. These are all ODE solvers.

File: kisao.owl (hunk 2 above)
Comment (luciansmith) [reply kisao.owl:14841]: Done!

**Comment by luciansmith** [comment]
OK, changes made!  Thanks again for the comments.

**Comment by luciansmith** [comment]
OK, going ahead and merging this.  Do let me know if anyone needs something more.

**Pull request merged**